functional/churn/playlist.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../TKG/resources/playlist.xsd">
+	<test>
+		<testCaseName>churn_1m_allGCs</testCaseName>
+		<command> 
+            export OTOOL_JDK_VERSION="$(JDK_VERSION)" ; \
+            export JREJDK="jdk" ; \
+            export OTOOL_garbageCollector="ALL" ; \
+            export DURATION="60" ; \
+            export TMPRESULTS="$(Q)$(REPORTDIR)$(D)report$(Q)"; \
+            cd $TMPRESULTS ; \
+              bash "$(TEST_ROOT)$(D)functional$(D)churn$(D)churn$(D)run.sh"; \
+            $(TEST_STATUS)
+		</command>
+		<levels>
+			<level>dev</level>
+		</levels>
+		<platformRequirementsList>
+			<platformRequirements>os.linux</platformRequirements>
+			<platformRequirements>os.win</platformRequirements>
+			<platformRequirements>os.osx</platformRequirements>
+		</platformRequirementsList>
+		<groups>
+			<group>system</group>
+		</groups>
+	</test>
+	<test>
+		<testCaseName>churn_5h_allGCs</testCaseName>
+		<command> 
+            export OTOOL_JDK_VERSION="$(JDK_VERSION)" ; \
+            export JREJDK="jdk" ; \
+            export OTOOL_garbageCollector="ALL" ; \
+            export DURATION="18000" ; \
+            export TMPRESULTS="$(Q)$(REPORTDIR)$(D)report$(Q)"; \
+            cd $TMPRESULTS ; \
+              bash "$(TEST_ROOT)$(D)functional$(D)churn$(D)churn$(D)run.sh"; \
+            $(TEST_STATUS)
+		</command>
+		<levels>
+			<level>dev</level>
+		</levels>
+		<platformRequirementsList>
+			<platformRequirements>os.linux</platformRequirements>
+			<platformRequirements>os.win</platformRequirements>
+			<platformRequirements>os.osx</platformRequirements>
+		</platformRequirementsList>
+		<groups>
+			<group>system</group>
+		</groups>
```

Comment:
```
<vendors>
        <vendor>eclipse</vendor>
        <vendor>redhat</vendor>
</vendors>
```

Reply:
I will add this element, But I think the generic openjdk should be for sure here.  I now recall, there may be issue with shenandoah on jdk8, but I would rather workaround it in CHURN itself, then exclude generic oepnjdk. Thoughts?

Reply:
I had made the listing of GCs dynamic: https://github.com/rh-openjdk/churn/commit/17f1b833249028f9efee13a4411f4b39ed592b0a

it is still based on known values, but at least will not enforce not-existing GC. Any advice on enabling more vendors? (compelltyh up to you :) tyvm!!)